### DISCUSSION TITLE
Annotation Editor Layer doesnt rendering in correct sizes

### DISCUSSION THREAD

**Discussion by ademyalcin27** [mozilla/pdf.js#18140] [Q&A]
Attach (recommended) or Link to PDF file here:
When I wanted to use the annotation editor layer I realized it doesn't render the correct size of the page. you will see extra space right and bottom its not covering page 100% it looks bigger than page.

Configuration:
- Web browser and its version: chrome Version 125.0.6422.61 (Official Build) (x86_64)
- Operating system and its version: Mac OS
- PDF.js version: 4.2.67
- Is a browser extension:

Steps to reproduce the problem:
1.
2.

What is the expected behavior? (add screenshot)
![Screenshot 2024-05-21 at 14 29 05](https://github.com/mozilla/pdf.js/assets/5026172/17423c30-842f-4818-ab30-b8202c5d1dc9)

What went wrong? (add screenshot)

![Screenshot 2024-05-21 at 14 26 09](https://github.com/mozilla/pdf.js/assets/5026172/eeddac3f-efcb-43b0-b255-7a4ccc3cb694)

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by Snuffleupagus** [comment]
> Attach (recommended) or Link to PDF file here:

This part is *always required* in order for an issue to be actionable/valid.

Also, please see https://github.com/mozilla/pdf.js/blob/master/.github/CONTRIBUTING.md (emphasis mine):
> If you are developing a custom solution, first check the examples at https://github.com/mozilla/pdf.js#learning and search existing issues. If this does not help, **please prepare a short well-documented example that demonstrates the problem and make it accessible online** on your website, JS Bin, GitHub, etc. before opening a new issue or contacting us in the Matrix room -- **keep in mind that just code snippets won't help us troubleshoot the problem**.

**Comment by ademyalcin27** [comment]
> > Attach (recommended) or Link to PDF file here:
> 
> This part is _always required_ in order for an issue to be actionable/valid.
> 
> Also, please see https://github.com/mozilla/pdf.js/blob/master/.github/CONTRIBUTING.md (emphasis mine):
> 
> > If you are developing a custom solution, first check the examples at https://github.com/mozilla/pdf.js#learning and search existing issues. If this does not help, **please prepare a short well-documented example that demonstrates the problem and make it accessible online** on your website, JS Bin, GitHub, etc. before opening a new issue or contacting us in the Matrix room -- **keep in mind that just code snippets won't help us troubleshoot the problem**.

I will provide some of code tomorrow or soon
